src/Servers/Kestrel/Transport.MsQuic/src/Internal/MsQuicStream.cs

```diff
@@ -390,20 +401,45 @@ public unsafe void EnableReceive()
             SetParam(QUIC_PARAM_STREAM.RECEIVE_ENABLED, buffer);
         }
 
+        private unsafe long GetStreamId()
+        {
+            var byteArr = new byte[8]; // size of long
```

Comment:
Nit:
```suggestion
            var byteArr = new byte[sizeof(long)];
```
(results in same code).